Hearthstone Deck Tracker/Controls/ElementSorterItem.xaml.cs

```diff
@@ -21,7 +21,33 @@ public partial class ElementSorterItem
 		public ElementSorterItem(string name, bool isChecked, Action<bool> setConfigValue, bool isPlayerList)
 		{
 			InitializeComponent();
-			CheckBox.Content = name;
+			switch (name)
+			{
+				case "Deck Title":
+					CheckBox.Content = Utility.LocUtil.Get("Options_Overlay_SortedElement_DeckTitle");
+					break;
+				case "Cards":
+					CheckBox.Content = Utility.LocUtil.Get("Options_Overlay_SortedElement_Cards");
+					break;
+				case "Card Counter":
+					CheckBox.Content = Utility.LocUtil.Get("Options_Overlay_SortedElement_CardCount");
+					break;
+				case "Fatigue Counter":
+					CheckBox.Content = Utility.LocUtil.Get("Options_Overlay_SortedElement_FatigueCount");
+					break;
+				case "Draw Chances":
+					CheckBox.Content = Utility.LocUtil.Get("Options_Overlay_SortedElement_DrawChances");
+					break;
+				case "Wins":
+					CheckBox.Content = Utility.LocUtil.Get("Options_Overlay_SortedElement_Wins");
+					break;
+				case "Win Rate":
+					CheckBox.Content = Utility.LocUtil.Get("Options_Overlay_SortedElement_WinRate");
+					break;
+				default:
+					CheckBox.Content = name;
+					break;
+			}
```

Comment:
That makes sense, I expected it more to basically reset when changing the language (or possibly crash)  because it uses different strings.